plugins/@grouparoo/sqlite/src/lib/table-import/getSampleRows.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  GetSampleRowsMethod,
+  DataResponseRow,
+} from "@grouparoo/app-templates/dist/source/table";
+
+export const getSampleRows: GetSampleRowsMethod = async ({
+  connection,
+  tableName,
+}) => {
+  const query = `SELECT * FROM ${tableName} WHERE id IN (SELECT id FROM ${tableName} ORDER BY RANDOM() LIMIT 10)`;
```

Comment:
Other databases got really slow when `ORDER BY RANDOM()` was used.  Is SQLite OK?

Reply:
I tested with 1k rows and the performance didn't bother me. Would you prefer I go the Postgres route and just pick the first 10?

Reply:
I don't think it matters at the moment - but it really was a bottleneck for large MySQL/Postgres databases, so I wanted to keep an eye out for it